cranelift/codegen/src/cursor.rs

```diff
@@ -759,11 +759,7 @@ impl<'c, 'f> ir::InstInserterBase<'c> for &'c mut EncCursor<'f> {
         &mut self.func.dfg
```

Comment:
Can `EncCursor` be removed entirely? It doesn't update encodings anymore.

Reply:
Sounds like yes!